validator/rules/config.go

```diff
@@ -58,6 +59,9 @@ func ValidatorConfigToRuleConfig(validatorConfig *ValidatorConfig, ruleConfig *R
 				Verbose: defaultVerbose,
 				Fields:  map[string]bool{"Use": true, "Short": true, "Long": true, "Example": true},
 			},
+			UseMatches: UseMatches{
+				Regexp: `^[^-_]+$`,
```

Comment:
Some rules might not need defaults. Is that default you would like to enforce?

Reply:
yes this is default, we can find some good default value, or can leave it empty

Reply:
Maybe a default that checks for line breaks or EOF. Something basic that everyone wouldn't want.

Reply:
kept if empty for now!